check.sh

```diff
@@ -0,0 +1,13 @@
+echo "Cloning opentelemetry-proto folder"
+svn checkout https://github.com/open-telemetry/opentelemetry-proto/trunk/opentelemetry
```

Comment:
Is there a reason you chose to use subversion for this checkout rather than git?  I think most people would prefer git in this scenario.

I also think we should validate that the script has the `git` binary available to the user.  If it doesn't, we should message appropriately.  We will probably run this script in a docker container, so we should be sure to choose an image that has git available.

Reply:
When we use git, we'll also want to reference https://github.com/open-telemetry/opentelemetry-proto rather than https://github.com/open-telemetry/opentelemetry-proto/trunk/opentelemetry